docs/getting-started/configuration.md

```diff
@@ -0,0 +1,25 @@
+---
+id: configuration
+title: Configuration
+sidebar_label: Configuration
+---
+
+> If you are using Docker container, all important configuration is set within the container.
+
+### Laravel variables
+
+* **APP_ENV** - this value defines environment used on the server. It can have two different values: local that prepares the application for testing on the development server, and production that will fully optimize the application for the mass usage in production.
+* **APP_DEBUG** - this field determines whether debugging tools will be ON or OFF. Keep it in sync with APP_ENV, where if `APP_ENV=production` then always use `APP_DEBUG=false`, otherwise `APP_DEBUG=true`.
+* **APP_KEY** - this is a random key generated using command `php artisan key:generate`. It is used in the process of generating and verifying passwords - so if you ever decide to change this value after the initial setup, it will cause all passwords generated by Laravel to be no longer valid.
+* **APP_URL** -  this variable should contain your application’s address. It’s used to generate urls in application.
+* **DB_…** - these six fields are used to configure a connection to your database. Most of the fields are self-explanatory. The only exception is “DB_CONNECTION”, where you should use “mysql”.
+* **CACHE_DRIVER** - should have value `file`, which is standard for CMS. Cache files will be stored in “storage/framework/cache”. Don’t forget that `www-data` user needs write permissions to that directory.
+* **SESSION_DRIVER** - should have value `file`, which is standard for CMS. Session files will be stored in “storage/framework/sessions”. Don’t forget that `www-data` user needs write permissions to that directory.
+* **SESSION_LIFETIME** - this value decides, how long will user's session last in minutes.
+
+### SIMPLOCMS variables
+
+* **SENTRY_DSN** - keep default value. It helps us to track issues in system and fix them.
```

Comment:
Asi bych potom tu defaultní hodnotu při releasu CMSka odstranil, chodily by nám tam i chyby, které nemusí být spojené s CMSkem, ale šablonou a dost by se to asi plevelilo.

Reply:
Zatím to schválně v dokumentaci nechám, abychom na to nezapomněli :)